src/components/IntegrationsView/IntegrationDocker/DockerContainerCard.tsx

```diff
@@ -0,0 +1,248 @@
+                    <DropdownMenu.Content size="1" side="bottom" align="end">
+                      {DOCKER_ACTIONS.map((dockerActionButton) => {
+                        const action = dockerActionButton.action;
+                        const label =
+                          currentContainerAction?.action ===
+                            dockerActionButton.action &&
+                          currentContainerAction.container === container.name
+                            ? dockerActionButton.loadingLabel
+                            : dockerActionButton.label;
+
+                        return (
+                          <DropdownMenu.Item
+                            key={dockerActionButton.label}
+                            disabled={isDockerActionButtonDisabled(
+                              container,
+                              action,
+                            )}
+                            onClick={() =>
+                              handleClickOnAction({
+                                container: container.name,
+                                action: dockerActionButton.action,
+                              })
+                            }
+                            color={
+                              dockerActionButton.action !== "start"
+                                ? "red"
+                                : undefined
+                            }
+                          >
+                            {label}
+                          </DropdownMenu.Item>
+                        );
+                      })}
+                      <DropdownMenu.Separator />
+                    </DropdownMenu.Content>
+                  </DropdownMenu.Root>
+                </Flex>
+              </Flex>
+            </Collapsible.Trigger>
+          </Flex>
+          <Collapsible.Content
+            style={{
+              marginTop: "1.15rem",
+            }}
+          >
+            <Flex direction="column" gap="4">
+              <Reveal defaultOpen={Object.values(container.env).length < 9}>
+                <Flex direction="column" gap="2" align="start">
+                  <Heading as="h5" size="4">
+                    Environment variables
+                  </Heading>
+                  <DataList.Root size="1">
+                    {Object.values(container.env).map((value) => {
+                      const [variableName, variableValue] = value.split("=");
+                      if (variableValue.startsWith("http")) {
+                        return (
+                          <DataList.Item key={variableName}>
+                            <DataList.Label>{variableName}: </DataList.Label>
+                            <DataList.Value>
+                              <TruncateRight
+                                size="1"
+                                style={{
+                                  width: "100%",
+                                }}
+                                title={variableValue}
```

Comment:
Is there a difference from having the title on the link?

Reply:
Nope, there is no difference. The only reason why I made titles for `<TruncateRight />` is that in URL scenario we have links, but otherwise we have `<Code />` blocks with variable values, I thought that it might be better to place a title upper on the wrapper for the actual content